prov/gni/Makefile.am

```diff
@@ -36,7 +36,8 @@ _gni_files = \
 	prov/gni/src/gnix_tags.c \
 	prov/gni/src/gnix_util.c \
 	prov/gni/src/gnix_vc.c \
-	prov/gni/src/gnix_wait.c
+	prov/gni/src/gnix_wait.c \
+	prov/gni/src/gnix_atomic.c
```

Comment:
in the spirit of maintaining alphabetical order could you move this up to above gnix_av.c?